qa/common/infer_util.py

```diff
@@ -55,14 +55,21 @@ def _range_repr_dtype(dtype):
         return np.int32
     return dtype
 
+def _match_registered(registered_shm_regions, shm_name, byte_size):
```

Comment:
This can return True, False, or None.

If you remove the `else`, the return type is saner.  (Always bool)

Reply:
Done